README.md

```diff
@@ -182,6 +182,51 @@ related:
 ```
 
 
+## components.yaml
```

Comment:
> This will eventually be used to link back to said documentation from both nuget usages, samples and articles.

I didn't know that, would be good to explain in the very beginning what's that for (currently, and potentially in the future).